geminimd/commands.py

```
@@ -0,0 +1,401 @@
+            download_attachments=options.download_attachments,
+            drive=drive,
+            force=options.force,
+            dry_run=options.dry_run,
+        )
+        if not options.dry_run and context.modified_time:
+            mtime = parse_rfc3339_to_epoch(context.modified_time)
+            if mtime:
+                try:
+                    os.utime(md_path, (mtime, mtime))
+                except Exception:
+                    pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)